ansible/roles/host-lets-encrypt-certs-certbot/tasks/main.yml

```diff
@@ -9,7 +9,7 @@
   block:
     - name: Verify if AWS Credentials exist on the host
       stat:
-        path: "/home/{{ _certbot_user }}/.aws/credentials"
+        path: "{{ ('~' +_certbot_user + '/.aws/credentials') | expanduser }}"
```

Comment:
I think `expanduser` is a mistake here. I believe it will evaluate the current user, `~` , on localhost because all filters run on localhost.

Reply:
you are right, let me solve it